jenkins/debian/debos/scripts/crush.sh

```diff
@@ -49,7 +49,7 @@ rm -rf etc/dpkg
 # Drop directories not part of ostree
 # Note that /var needs to exist as ostree bind mounts the deployment /var over
 # it
-rm -rf var/* opt srv share
+rm -rf var/* srv share
```

Comment:
Well, that's fine as there's nothing in `/opt` by default unless something comes along and installs a test suite.

So I've looked at how the other ones are set up (v4l2, igt) and everything is installed in `/usr` instead.  I guess it's a bit more correct to install things built from source and not managed by the distro in `/opt`, so that's fine.

It would still be nice to follow the same approach in all the rootfs variants, i.e. update the other ones to install their tests in `/opt` and update the test plan templates accordingly - but that's outside of the scope for this PR.